.github/workflows/diagram-cache.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright DB Netz AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: update_capella_diagram_cache
+
+on: push
+
+jobs:
+  update_capella_diagram_cache:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v3
+        - name: Set up Python 3.11
+          uses: actions/setup-python@v3
+          with:
+            python-version: 3.11
+        - name: Install capellambse
+          run:
+            pip install "capellambse[cli]@git+https://github.com/DSD-DBS/py-capellambse.git@$CAPELLAMBSE_REVISION"
+        - name: Create diagram cache
+          run:
+            python -m capellambse.diagram_cache -m ${{ env.ENTRYPOINT }} -o ./diagram_cache --index --docker ghcr.io/dsd-dbs/capella-dockerimages/capella/base:{VERSION}-${{ env.CAPELLA_DOCKER_IMAGES_DROPINS }}-dropins-${{ env.CAPELLA_DOCKER_IMAGES_REVISION }}
```

Comment:
And to make it more readable, please make a multi-line string out of it: https://yaml-multiline.info/ :)